aiidalab/app.py

```diff
@@ -653,6 +744,12 @@ def __init__(self, app, with_version_selector=False):
         self.update_button = ipw.Button(description='Update', disabled=True)
         self.update_button.on_click(self._update_app)
 
+        self.install_kernel_button = ipw.Button(description='Install kernel', disabled=True)
+        self.install_kernel_button.layout.visilibity = 'hidden'
+        self.install_kernel_button.button_style = 'success'
+        self.install_kernel_button.tooltip = 'Install the app-specific Python and Jupyter kernel and app dependencies.'
+        self.install_kernel_button.on_click(self._install_kernel)
+
```

Comment:
This all might be packed in `ipw.Button(..)`

Reply:
Done.